Modelica/Electrical/Analog/Interfaces.mo

```diff
@@ -301,7 +301,7 @@ The reason could be that
   partial model ConditionalHeatPort
     "Partial model to include a conditional HeatPort in order to describe the power loss via a thermal network"
 
-    parameter Boolean useHeatPort = false "=true, if heatPort is enabled"
+    parameter Boolean useHeatPort = false "Set =true, if heatPort is enabled"
```

Comment:
There is no `Set`. The old version was perfectly fine.

Reply:
Yes, there ist no Set. The sentence means that the parameter useHeatPort should be set to true.
The old versiov seemed not to be fine since it was marked in #2636. It was said there, that the description string should start with upper case. I also likes the old version better. For me it is fine to go back to the old version